addons/l10n_es_edi_tbai/data/template_invoice.xml

```diff
@@ -0,0 +1,281 @@
+                </IDDestinatario>
+            </Destinatarios>
+            <VariosDestinatarios t-if="is_emission">N</VariosDestinatarios>
+            <!-- Odoo does not support multi-recipient invoices (TBAI does)-->
+            <EmitidaPorTercerosODestinatario t-if="is_emission" t-out="thirdparty_or_recipient"/>
+        </template>
+
+        <template id="template_invoice_factura">
+            <t t-set="is_simplified" t-value="invoice._is_l10n_es_tbai_simplified()"/>
+            <CabeceraFactura>
+                <SerieFactura t-out="invoice.l10n_es_tbai_sequence"/>
+                <NumFactura t-out="invoice.l10n_es_tbai_number"/>
+                <t t-if="is_emission">
+                    <FechaExpedicionFactura t-out="datetime_strftime(datetime_now, '%d-%m-%Y')"/>
+                    <HoraExpedicionFactura t-out="datetime_strftime(datetime_now, '%H:%M:%S')"/>
+                    <FacturaSimplificada t-out="'S' if is_simplified else 'N'"/>
+                </t>
+                <FechaExpedicionFactura t-else="" t-out="datetime_strftime(invoice.l10n_es_tbai_registration_date, '%d-%m-%Y')"/>
+                <t t-if="is_refund">
+                    <FacturaEmitidaSustitucionSimplificada t-out="'S' if (is_simplified and recipient) else 'N'"/>
+                    <FacturaRectificativa>
+                        <Codigo t-out="credit_note_code"/>
+                        <Tipo>I</Tipo>
+                        <!-- TODO also allow credit note Tipo 'S' (optional)
+                        <ImporteRectificacionSustitutiva>
+                            <BaseRectificada>180.00</BaseRectificada>
+                            <CuotaRectificada>20.21</CuotaRectificada>
+                        </ImporteRectificacionSustitutiva> -->
+                    </FacturaRectificativa>
+                    <FacturasRectificadasSustituidas>
+                        <IDFacturaRectificadaSustituida>
+                            <!-- TODO: support issuing a single credit note for multiple invoices (optional) -->
+                            <SerieFactura t-out="credit_note_invoice.l10n_es_tbai_sequence"/>
+                            <NumFactura t-out="credit_note_invoice.l10n_es_tbai_number"/>
+                            <FechaExpedicionFactura t-out="datetime_strftime(credit_note_invoice.l10n_es_tbai_registration_date, '%d-%m-%Y')"/>
+                        </IDFacturaRectificadaSustituida>
+                    </FacturasRectificadasSustituidas>
+                </t>
+            </CabeceraFactura>
+            <DatosFactura t-if="is_emission">
+                <DescripcionFactura t-out="invoice.invoice_origin or 'manual'"/>
+                <DetallesFactura>
+                    <IDDetalleFactura t-foreach="invoice_lines" t-as="line_value">
+                        <t t-set="line" t-value="line_value['line']"/>
+                        <t t-set="discount" t-value="line.price_subtotal / (1 - line.discount / 100) * line.discount / 100"/>
```

Comment:
why not from Python?

Reply:
Please review the v16 PR here
https://github.com/odoo/odoo/pull/109907